x-pack/plugins/lens/public/metric_visualization/auto_scale.tsx

```diff
@@ -85,6 +90,12 @@ export class AutoScale extends React.Component<Props, State> {
               style={{
                 transform: `scale(${scale})`,
               }}
+              className={classNames('lnsMetricExpression__containerScale', {
+                alignLeft: textAlign === 'left',
+                alignRight: textAlign === 'right',
+                alignCenter: textAlign === 'center',
+                [`titleSize${(size ?? 'xl').toUpperCase()}`]: true,
```

Comment:
Any reason why the default size isn't handled on the `toExpression` level like position and alignment?

Reply:
removed default here because we use default font size in class `lnsMetricExpression__value` and don't needed to add this class if size is not defained.